tests/unit/routes/test_person_router.py

```diff
@@ -112,3 +117,85 @@ def test_with_patients(self, client):
             "person_reference_id": str(person.reference_id),
             "patient_reference_ids": [str(pat1.reference_id), str(pat2.reference_id)],
         }
+
+
+class TestMergePersonClusters:
+    def testMergePersonClustersSuccess(self, client):
+        person1 = models.Person()
+        patient1 = models.Patient(person=person1, data={})
+
+        person2 = models.Person()
+        patient2 = models.Patient(person=person2, data={})
+
+        client.session.add_all([patient1, patient2])
+        client.session.flush()
+
+        response = client.post(
+            f"/person/{person1.reference_id}/merge/",
+            json={"person_reference_ids": [str(person2.reference_id)]},
+        )
+
+        assert response.status_code == 200
+        assert response.json()["person_reference_id"] == str(person1.reference_id)
+
+    def testInvalidPersonIdType(self, client):
+        response = client.post(
+            "/person/123/merge/",
+            json={"person_reference_ids": [str(uuid.uuid4())]},
+        )
+        assert response.status_code == 422
+
+    def test1InvalidPersonId(self, client):
+        # Tests that a 422 will be raised if a single invalid person id is passed
+        person1 = models.Person()
+        patient1 = models.Patient(person=person1, data={})
+
+        person2 = models.Person()
+        patient2 = models.Patient(person=person2, data={})
+
+        client.session.add_all([patient1, patient2])
+        client.session.flush()
+
+        response = client.post(
+            f"/person/{person1.reference_id}/merge/",
+            json={"person_reference_ids": [str(person2.reference_id), str(uuid.uuid4())]},
+        )
+        assert response.status_code == 422
+
+    def testPersonIdsNottheSame(self, client):
+        # Test that a 422 will be raised if the merge_into_id is in the person_reference_ids
+        person1 = models.Person()
```

Comment:
```suggestion
        person1 = models.Person(reference_id=uuid.uuid4())
```
By default, the reference_id is `null`.  If we persist that, SQLAlchemy will [create a default value](https://github.com/CDCgov/RecordLinker/blob/main/src/recordlinker/models/mpi.py#L22) for us.  Alternatively, we can just initialize the object with a value and skip the `session.add`, `session.flush` operations.